core/src/ibc/connection_03/manager.rs

```diff
@@ -24,6 +24,11 @@ use rlp::{Encodable, Rlp};
 #[derive(Default)]
 pub struct Manager {}
```

Comment:
no context?

Reply:
Let' add context at the same time in another PR. I was creating this PR before the discussion.

Reply:
Ok no problem